crates/holochain/src/core/sys_validate.rs

```diff
@@ -211,7 +211,7 @@ pub async fn check_spam(_header: &Header) -> SysValidationResult<()> {
 
 /// Check previous header timestamp is before this header
 pub fn check_prev_timestamp(header: &Header, prev_header: &Header) -> SysValidationResult<()> {
-    if header.timestamp() >= prev_header.timestamp() {
+    if header.timestamp() > prev_header.timestamp() {
```

Comment:
I am just seeing this and thinking this will re-break windows build I think... :s 
or maybe just break windows usage and not the build @ddd-mtl would know. 
He made this change not long ago and it was approved and merged
https://github.com/holochain/holochain/pull/880#issue-691684169

Reply:
@Connoropolous @ddd-mtl see the other change in this PR that compares the current chain head and increments the time by 1 nanosecond if needed to ensure it always meets this strict inequality where the header builder is used

Reply:
for context, when working with countersigning sessions the clock can be _very_ not monotonic relative to the chain, on the order of several seconds, so simply allowing 1 extra nano of leeway here in the validation may have fixed issues with local system times, but isn't enough when times are shared with other agents - that's why i shifted the logic to the header builder

ensuring the monotonicity was a request from @artbrock to remove a potential attack/bug surface, even if we don't have a specific example of how this could be abused yet

Reply:
also yes, existing data will start failing validation, this is a breaking change

Reply:
also also, i don't think this was ever a real fix because according to the rust docs, time can go _backwards_ not just appear stationary:

https://doc.rust-lang.org/std/time/struct.SystemTime.html

> Distinct from the Instant type, this time measurement is not monotonic. This means that you can save a file to the file system, then save another file to the file system, and the second file has a SystemTime measurement earlier than the first. In other words, an operation that happens after another operation in real time may have an earlier SystemTime!

maybe this points to using `Instant` rather than `SystemTime`... but that still wouldn't fix the networked situation

Reply:
Ok gotcha

Reply:
I believe this should be fine, as it should guarantee monotonicity of the header timestamps.
I'd avoid using `Instant` as it has a different behavior on Windows.